tests/Test/Dscp/Crypto/Serialise.hs

```diff
@@ -0,0 +1,12 @@
+module Test.Dscp.Crypto.Serialise where
```

Comment:
Please also add positive/negative tests for the encryption itself (not for the binary instances only).

Reply:
They are already added in previous PR, look at the `Test.Dscp.Crypto.Encrypt` module.

Reply:
Aha! Alright then.